src/sys/unix/kqueue.rs

```diff
@@ -35,7 +36,7 @@ impl Selector {
         Ok(Selector {
             id: id,
             kq: kq,
-            changes: RefCell::new(KeventList(Vec::new())),
+            changes: RefCell::new(KeventList(Vec::with_capacity(32))),
```

Comment:
I feel like a better option here would be to use a statically sized ring-like buffer, but that would increase size of Selector significantly, which I’m not sure is desired.